eduvpn/ui/stats.py

```diff
@@ -0,0 +1,138 @@
+import os
+import logging
+import time
+from datetime import timedelta
+
+from ..nm import get_iface, get_ipv4, get_ipv6, get_timestamp
+
+logger = logging.getLogger(__name__)
+
+LINUX_STATS_FILE = "/sys/class/net/{iface}/statistics/{filename}"
+
+
+class NetworkStats():
+    def __init__(self, interval):
+        self.iface = None
+        self.interval = interval
+        self.addresses = {}
+        self.files = {}
+        self.start_bytes = {}
+        self.timestamp = None
+        self.default_text = "N/A"
```

Comment:
i think we should use the translation helper function here

Reply:
I will try it! Does the rest of the ui translate too?

Reply:
yep!